packages/zoe/src/zoeService/zoeSeat.js

```diff
@@ -7,8 +7,8 @@ import { makePromiseKit } from '@endo/promise-kit';
 import { NonNullish } from '@agoric/internal';
 
 import { satisfiesWant } from '../contractFacet/offerSafety.js';
-import '../types-ambient.js';
-import '../internal-types.js';
+// import '../types-ambient.js';
+// import '../internal-types.js';
```

Comment:
Why leave these around? Are they worth an XXX comment?

Reply:
thanks. not intentional, I'll remove them